.env

```diff
@@ -1,3 +1,3 @@
 STAGE_URL=https://www.kaggle.com/
-EMAIL=testmail@mail.com
-PASSWORD=nwiefhwheffwef
\ No newline at end of file
+EMAIL=woreso1080@ikumaru.com
```

Comment:
It'd be better to add this file to .gitignore file in your local project - just not to make credentials visible to everyone